features/logistics/variable-measurement-ratios.md

```diff
@@ -0,0 +1,40 @@
+# Variable (Dynamic) Measurement Ratios
+
+## Overview
+
+Some industries trade products the quantity of which **may vary to some degree** for each individual piece of the product that is bought, processed, and sold. 
```

Comment:
We need (small) example in the first sentence. For example:
"...products (like potatoes, iron bars, fuel and others), the quantity of which..."

Reply:
Added some products so readers can identify them better.